gazelle/README.md

```diff
@@ -157,11 +184,29 @@ Next, all source files are collected into the `srcs` of the `py_library`.
 Finally, the `import` statements in the source files are parsed, and
 dependencies are added to the `deps` attribute.
 
-### Tests
+### Unit Tests
```

Comment:
Gazelle now supports generating 1 target per test it finds when `__test__.py` or `__test__` target doesn't exist.

Reply:
How would your recommend saying that in more detail?

Reply:
Something like:

```
Gazelle will detect test files that meet the `test_*.py` and `*_test.py` patterns and add one `py_test` per test detected. If you want to group multiple test files into a single `py_test`, use the `__test__.py` or `__test__`, bla bla bla bla bla
```